cmd/down.go

```diff
@@ -17,23 +21,37 @@ var downCmd = &cobra.Command{
 		var err error
 		c.GracePeriod, err = cmd.Flags().GetInt64("grace-period")
 		if err != nil {
-			return err
+			return fmt.Errorf("Can't get --grace-period flag: %v", err)
 		}
 
 		c.ClientPool, c.Discovery, err = restClientPool()
 		if err != nil {
-			return err
+			return fmt.Errorf("Can't get Kubernetes client: %v", err)
+		}
+		// validate k8s version
+		version, err := utils.FetchVersion(c.Discovery)
+		if err != nil {
+			return fmt.Errorf("Can't verify Kubernetes version: %v", err)
+		}
+		if version.Major <= 1 && version.Minor < 7 {
+			fmt.Println("Warning: Kubernetes with RBAC enabled (v1.7+) is required to run Kubeapps")
+			os.Exit(0)
```

Comment:
the exit code should be different than 0 to return an error. Also, why exiting instead of returning the error?

Reply:
Actually it isn't an error. We just want to send a warning message and quit the installer gently.

Reply:
but it should be an error, I mean if I want to deploy kubeapps and the command doesn't succeed the exit code shouldn't be 0

Reply:
I agree with @andresmgot , this should be considered an error

Reply:
Okie I turned to an error message in the follow-up commit. Please @arapulido @andresmgot take a look